docs/static/schema.json

```
@@ -134,7 +141,7 @@
           "method": {
             "description": "Defines which method is used to check the task is up-to-date. `timestamp` will compare the timestamp of the sources and generates files. `checksum` will check the checksum (You probably want to ignore the .task folder in your .gitignore file). `none` skips any validation and always run the task.",
             "type": "string",
-            "enum": ["none", "checksum", "timestamp"],
+            "enum": [ "none", "checksum", "timestamp" ],
```

I mentioned this on another PR IIRC, but I'd revert this style change as it's not common IMO

```suggestion
            "enum": ["none", "checksum", "timestamp"],
```

---

Whoops. I'm not doing this on purpose, I swear 🤣